addons/mail/static/src/messaging/component/composer/composer.js

```diff
@@ -169,9 +164,7 @@ class Composer extends Component {
      */
     async _postMessage() {
         // TODO: take suggested recipients into account
-        await this.storeDispatch('postMessage', this.props.composerLocalId, {
-            isLog: this.props.isLog,
-        });
+        await this.composer.postMessage({ isLog: this.props.isLog });
```

Comment:
`isLog` tells whether the message post on this composer should be treated as a log or not.
Not in composer entity because there's a single composer per thread, and we don't want to share this mode for each viewer of composer